tests/featureWriters/markFeatureWriter_test.py

```diff
@@ -1415,6 +1415,29 @@ def test_quantize(self, testufo):
             """
         )
 
+    def test_extra_substitutions(self, FontClass):
+        dirname = os.path.dirname(os.path.dirname(__file__))
+        fontPath = os.path.join(dirname, "data", "Alternates-Regular.ufo")
+        testufo = FontClass(fontPath)
+        generated = self.writeFeatures(
+            testufo,
+            compiler=FeatureCompiler(
+                testufo,
+                extraSubstitutions={
+                    "uuMatra-oriya": set(["uuMatra-oriya.BRACKET.varAlt01"]),
```

Comment:
you could use set literals `{"uuMatra-oriya.BRACKET.varAlt01"}` instead of `set([...])`